mcux/drivers/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 #
-# Copyright (c) 2018, NXP
+# Copyright (c) 2018-2020, NXP
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
-if("${CONFIG_SOC_FAMILY}" STREQUAL "nxp_imx")
+if("${CONFIG_SOC}" STREQUAL "mimxrt685s_cm33")
```

Comment:
Can we change this from CONFIG_SOC to based on something like SOC_SERIES?

Reply:
Done